src/odemis/driver/test/actuator_test.py

```diff
@@ -548,6 +548,49 @@ def test_metadata(self):
 
         testing.assert_pos_almost_equal(stage.speed.value, {"x": 10e-6, "y": 20e-6})
 
+    def test_get_axes_range(self):
+        """
+        Test the range calculation in ConvertStage
+        """
+        # Test based on given range of Zeiss stage
+        dependency = simulated.Stage("stage", "test", axes=["y", "m"],
+                                     ranges={"y": [0.e-3, 100.e-3], "m": [0.e-3, 13.e-3]})
+
+        # Transform into Meteor stage , rotated by 26 degrees --> 0.4537856055185 rad
+        tilt = math.radians(26)
+        stage = ConvertStage("conv", "stage", {"orig": dependency},
+                             axes=["y", "m"], rotation=tilt)
+        first_value = 2e-03
+        second_value = 0
```

Comment:
Can you leave a short comment as to what these values are supposed to represent?